Unable to pack Microsoft.VisualBasic.Forms

* .NET Core Version: master
* Have you experienced this same bug with .NET Framework?: No

<!-- Read https://github.com/dotnet/winforms/blob/master/Documentation/issue-guide.md -->

**Problem description:**

The dev version is not flowing through to core-sdk, isn't getting bundled, and as a result isn't available to our early adopters.
After a discussion with @AdamYoblick and @dagood it was concluded that the flow is broken because we are unable to pack, due to a missing intellisense file for Microsoft.VisualBasic.Forms project.
```
C:\Development\winforms\.dotnet\sdk\5.0.100-alpha1-013788\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(198,5): 
error NU5019: File not found: 'C:\Development\winforms\artifacts\bin\Microsoft.VisualBasic.Forms\Debug\netcoreapp5.0\ref\Microsoft.VisualBasic.Forms.xml'. [C:\Development\winforms\pkg\Microsoft.Private.Winforms\Microsoft.Private.Winforms.csproj]

Build FAILED.
```

Microsoft.VisualBasic.Forms project is a recent addition, and thus its docs haven't been ingested into the docs project.

**Expected behavior:**

` > .\build -pack` executes without errors.

--- a/eng/packageContent.targets
+++ b/eng/packageContent.targets
@@ -19,6 +19,8 @@
 
   <PropertyGroup>
     <IntellisenseXmlFileSource>$(IntellisenseXmlDir)$(AssemblyName).xml</IntellisenseXmlFileSource>
+    <!-- TODO: remove when we get dotnet-api-docs_netcoreapp5.0 -->
+    <IntellisenseXmlFileSource Condition="!Exists('$(IntellisenseXmlFileSource)')">$([System.IO.Path]::ChangeExtension('$(TargetPath)', '.xml'))</IntellisenseXmlFileSource>
     
     <IntellisenseXml Condition="'$(ProduceReferenceAssembly)' == 'true' And '$(PackageAsRefAndLib)' != 'true'" >$([System.IO.Path]::ChangeExtension('$(TargetRefPath)', '.xml'))</IntellisenseXml>
     <IntellisenseXml Condition="'$(PackageAsRefAndLib)' == 'true'" >$([System.IO.Path]::ChangeExtension('$(TargetPath)', '.xml'))</IntellisenseXml>

--- a/src/Microsoft.VisualBasic.Forms/src/Microsoft.VisualBasic.Forms.vbproj
+++ b/src/Microsoft.VisualBasic.Forms/src/Microsoft.VisualBasic.Forms.vbproj
@@ -6,6 +6,7 @@
     <RootNamespace></RootNamespace>
     <LangVersion>15.0</LangVersion>
     <VBRuntime>None</VBRuntime>
+    <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
   </PropertyGroup>
 
   <ItemGroup>

